### PR TITLE
Prepare for wheel building

### PR DESCRIPTION
**Pull request by mmacata**
compare https://github.com/mundialis/actinia-gdi/pull/25.
This PR also adds a Dockerfile which builds this addon, creates a github release and publishes the wheel to github as a release asset.

### CODE REVIEW

**Comment by mmacata** [comment]
This PR also changes the Flask Version from `Flask==0.12.3` to `Flask>=0.12.3`. If requirements are installed after actinia_core, this is no problem, as it uses the existing Flask.

**Comment by mmacata** [comment]
Also see https://github.com/mundialis/actinia_satellite_plugin/pull/3

File: docker/create_release_with_asset.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#/bin/sh
```
Comment (neteler) [inline docker/create_release_with_asset.sh]:
```suggestion
#!/bin/sh
```

**Comment by neteler** [comment]
Besides the shebang typo it looks fine to me!

File: docker/create_release_with_asset.sh (hunk 1 above)
Comment (mmacata) [reply docker/create_release_with_asset.sh]: shebang fixed

**Comment by mmacata** [comment]
As reviewed, I'll merge now

**Pull request merged**